monix-reactive/jvm/src/test/scala/monix/reactive/compress/DeflateIntegrationTest.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2014-2020 by The Monix Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.reactive.compress
+
+import monix.eval.Task
+import monix.reactive.Observable
+import org.scalacheck.Prop
+
+object DeflateIntegrationTest extends BaseTestSuite with DeflateTestUtils {
+  private implicit def a[A]: Task[Boolean] => Prop =
+    _.runSyncUnsafe()
+
+  test("inflate(deflate(_)) == identity") {
```

Comment:
Nitpick but we usually describe it as `<->` - feel free to ignore the comment :D